docs/conf/helpop-full.conf.example

```diff
@@ -927,11 +930,39 @@ using their cloak when they quit.">
                     channel (requires stripcolor module).
  T                  Blocks /NOTICEs to the channel from users who are
                     not at least halfop (requires nonotice module).
-
- g <mask>           Blocks messages matching the given glob mask
-                    (requires chanfilter module).
- X <mode>           Makes channel operators immune to the specified
-                    restrictive mode (requires exemptchanops module).
+ X <mode>:<status>  Makes users of <status> or higher immune to the
+                    specified restrictive mode. For example: flood:h
+                    (requires exemptchanops module).
+Possible restrictive modes for +X are:
+
+auditorium-see (requires the auditorium module)
```

Comment:
The description should probably be after the module requirement.

Reply:
Also, the helpop file mostly uses uses "requires X module" whereas this uses "requires the X module". We should probably just decide on one. Probably the latter? it seems more grammatically correct to me.

Reply:
Did you mean "before" the module requirement, ie. like the others currently do? Which I now see would make more sense.  
I do agree, "requires the x module" sounds more correct as it's a specific module being referred to. I can go through and change the rest.

Reply:
Ideally it should be something like this:

```
wibble      Does something really important that is described here
                 (requires the wibbling module)
```